iFixit/react-commerce
# Add product list device navigation

Closes #8 

## Changelog

- Add a sub navigation to the product list page
- Move breadcrumbs into the subnavigation
- Add device navigation (parts, guides, answers)

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page (`/store/parts`)
3. Verify that "guides" and "answers" links lead to guides home and answers home respectively
4. Visit another product list page (e.g. `Vacuum Parts`) and verify that "guides" and "answers" links points to the device guides and answers pages
